run-test.sh

```diff
@@ -27,11 +27,11 @@ while (( "$#" )); do
   esac
 done
 
-docker run --rm \
+docker run --rm -it \
            -v "${DIR}/trex-scripts:/workspace/trex-scripts" \
            -v "${DIR}/tools:/workspace/tools" \
            -v "${DIR}/tmp:/tmp" \
-           -w /workspace \
+           -w /workspace/trex-scripts \
```

Comment:
wondering why we need to change the default workspace?

Reply:
I got this error otherwise:
```
./run-test.sh --server-addr 10.128.13.27 --trex-config trex-configs/4-ports-with-l2.yaml <test name>
/usr/bin/python3: No module named control
```

Reply:
I see, I think because the original container image includes additional `PYTHONPATH`.